src/cpu/aarch64/injectors/jit_uni_eltwise_injector.cpp

```diff
@@ -0,0 +1,2253 @@
+            do {
+                h->add_imm(h->x_tmp_vec[count++], h->X_SP, i * vlen,
+                        h->X_DEFAULT_ADDR);
+                i++;
+            } while (i < tail_vecs_to_preserve && count < h->x_tmp_vec_size);
+
+            for (int j = 0; j < count; j++)
+                h->ld1w(ZRegS(preserved_vec_idxs[idx_off + ii++]), p_all / T_z,
+                        ptr(h->x_tmp_vec[j]));
+        }
+    }
+
+    for (size_t i = 0; i < tail_vecs_to_preserve; ++i)
+        preserved_vec_idxs[idx_off + i] += tail_vecs_to_preserve;
+
+    if (save_state_) {
+        size_t i = 0;
+
+        while (i < tail_vecs_to_preserve) {
+            int count = 0;
+            int ii = i;
+            do {
+                h->add_imm(h->x_tmp_vec[count++], h->X_SP, i * vlen,
+                        h->X_DEFAULT_ADDR);
+                i++;
+            } while (i < tail_vecs_to_preserve && count < h->x_tmp_vec_size);
+
+            for (int j = 0; j < count; j++)
+                h->st1w(ZRegS(preserved_vec_idxs[idx_off + ii++]), p_all / T_z,
+                        ptr(h->x_tmp_vec[j]));
+        }
+
+        if (idx_off) {
+            h->sub_imm(h->X_SP, h->X_SP, idx_off * vlen, h->X_TMP_0);
+        }
+    }
+
+    assign_regs();
+}
+
+template <cpu_isa_t isa>
+void jit_uni_eltwise_injector_f32<isa>::injector_postamble() {
+    using namespace Xbyak_aarch64::util;
+    if (!save_state_) return;
+
+    size_t i = 0;
+
+    while (i < preserved_vecs_count) {
+        int count = 0;
+        int ii = i;
+        do {
+            h->add_imm(h->x_tmp_vec[count++], h->X_SP, i * vlen,
+                    h->X_DEFAULT_ADDR);
+            i++;
+        } while (i < preserved_vecs_count && count < h->x_tmp_vec_size);
+
+        for (int j = 0; j < count; j++)
+            h->ld1w(ZRegS(preserved_vec_idxs[ii++]), p_all / T_z,
+                    ptr(h->x_tmp_vec[j]));
+    }
+
+    if (preserved_vecs_count)
+        h->add_imm(h->X_SP, h->X_SP, preserved_vecs_count * vlen, h->X_TMP_0);
+
+    for (int i = aux_gprs_count() - 1; i >= 0; --i)
+        h->ldr(XReg(preserved_gpr_idxs[i]), pre_ptr(h->X_SP, 8));
+    h->ldr(x_table, pre_ptr(h->X_SP, 8));
+}
+
+template <cpu_isa_t isa>
+void jit_uni_eltwise_injector_f32<isa>::assign_regs() {
+    /* For translation of x64's memory operand instructions */
+    z_tmp = TRegS(static_cast<uint32_t>(preserved_vec_idxs[0]));
+
+    vmm_mask = TRegS(preserved_vec_idxs[1]);
+    vmm_aux0 = TRegS(preserved_vec_idxs[1]);
+    vmm_aux1 = TRegS(preserved_vec_idxs[2]);
+    vmm_aux2 = TRegS(preserved_vec_idxs[3]);
+    vmm_aux3 = TRegS(preserved_vec_idxs[4]);
+    vmm_aux4 = TRegS(preserved_vec_idxs[5]);
+    vmm_aux5 = TRegS(preserved_vec_idxs[6]);
+    vmm_aux6 = TRegS(preserved_vec_idxs[7]);
+    vmm_aux7 = TRegS(preserved_vec_idxs[8]);
+}
+
+// Uses injector masks objects: k_mask (>= avx512_common) or vmm_mask (<= avx2).
+// Stores a mask by applying cmpps on two inputs w/ a given predicate.
```

Comment:
Same as above, please modify comments to reflect the situation on aarch64, not on x64.